sourcecode/dist/dark-theme.js

```diff
@@ -41,7 +41,7 @@ let setLightTheme = () => {
     toggleClass("bg-darker", "bg-lighter");
     toggleClass("text-light", "text-dark");
     toggleClass("btn-light", "btn-dark");
-    toggleClass("table-light", "table-dark");
+    toggleClass("table-dark", "table-light");
```

Comment:
xD

Reply:
The thing is: I was just copy/pasting text and btn.
But text and buttons are the opposite due to contrast so... Yeah XD